lib/bottles.rb

```diff
@@ -1,2 +1,41 @@
 class TenGreenBottles
-end 
+
+
```

Comment:
Super pedantic, but I think there's an extra space here. Could be worth getting in the habit of using a linter ie. https://rubocop.readthedocs.io/en/latest/